.github/workflows/helm_release.yaml

```diff
@@ -0,0 +1,37 @@
+name: Release Charts
+
+on:
+  push:
+    branches:
+      - main
+
```

Comment:
Hey @joshsizer,

Mind changing the trigger from on push to `workflow_dispatch` instead?

I don't think I'd want every push to the repository's `main` branch to produce a release.

Reply:
@vyas-n A release is only produced when the version value in Chart.yaml changes on the main branch (not on every commit to main). Not sure if that changes your opinion at all, but if not I can definitely change it to `workflow_dispatch`.

Reply:
In this case, I'd still prefer that the workflow occurs on `workflow_dispatch`.

It'll keep the GitHub actions jobs a bit cleaner by only having executions when we are releasing, instead of each time it's pushed to the main branch and no-op'ing the commit.